charts/dft-backend/values.yaml

```diff
@@ -7,7 +7,7 @@ replicaCount: 1
 image:
   repository: ghcr.io/catenax-ng/tx-dft-backend
   pullPolicy: Always
-  tag: "1.8.1"
+  tag: "1.9.0"
```

Comment:
Consider leaving tag empty (`tag: ""`) to automatically use the appVersion defined in the `Charts.yaml` file. Then you don't need to adjust the tag each time you release a new helm chart version.

Reply:
Tag has been removed and tag is empty now.